packages/hub/routes/email-card-drop-requests.ts

```diff
@@ -140,7 +140,32 @@ export default class EmailCardDropRequestsRoute {
       return;
     }
 
-    let verificationCode = (crypto.randomInt(1000000) + '').padStart(6, '0');
+    let unclaimedButExisting = await this.emailCardDropRequestQueries.query({
+      emailHash: normalizedEmailHash,
+      ownerAddress: ctx.state.userAddress,
+    });
+
+    if (unclaimedButExisting.length > 0) {
+      let existingRequest = unclaimedButExisting[0];
+      let updatedEmailCardDropRequest = await this.emailCardDropRequestQueries.updateVerificationCode(
+        existingRequest.id,
+        generateVerificationCode()
+      );
+
+      await this.workerClient.addJob('send-email-card-drop-verification', {
+        id: updatedEmailCardDropRequest.id,
```

Comment:
Also, not related to this PR but wondering if this mutability of the verification code is why we don't simply pass the EOA + verification code into this job?

Reply:
I debated including it, do you think it’s preferable?